### PR TITLE
partitions feature for streams

### PR DESCRIPTION
**Pull request by vkudyushev**

### CODE REVIEW

File: python_liftbridge/python_liftbridge.py [hunk 1]
```diff
@@ -139,17 +141,20 @@ def _subscribe_request(self, stream):
                 stream=stream.name,
                 startPosition=stream.start_position,
                 startOffset=stream.start_offset,
+                partition=stream.partitions
             )
```
Comment (LaPetiteSouris) [inline python_liftbridge/python_liftbridge.py:145]: I think there is a confusion here.

In `CreateStreamRequest`, `partitions` parameter is to indicate how many partitions should the stream create. Refer [CreateStreamRequest](https://github.com/liftbridge-io/liftbridge-api/blob/edbf0f90bd66f6ecca3c6450a53e3bc27cac98f6/api.proto#L25)

But in `SubcribeRequest`, `partition` (not `partitions`) is to indicate to which partition should the subscription be sent to. Refer [SubscribeRequest](https://github.com/liftbridge-io/liftbridge-api/blob/edbf0f90bd66f6ecca3c6450a53e3bc27cac98f6/api.proto#L78)

Comment (vkudyushev) [reply python_liftbridge/python_liftbridge.py:145]: yep, argument name looks like a typo, but look at this example, extended code from README

```
from itertools import cycle
from random import choice

from python_liftbridge import Lift, Message, Stream, ErrStreamExists

client = Lift(ip_address='localhost:9293', timeout=5)
stream_name = 'foo'

try:
    client.create_stream(Stream(subject='foo', name=stream_name, partitions=5))
except ErrStreamExists:
    print('This stream already exists!')

cp = cycle([0, 1, 2, 3, 4])
for i in range(20):
    client.publish(Message(value=f'message {i}', stream=stream_name, partition=next(cp)))

pnum = choice(range(5))
for message in client.subscribe(
    Stream(
        subject='foo',
        name=stream_name,
        partitions=pnum
    ).start_at_earliest_received(),
):
    print(f"Received: '{message.value.decode()}' on partition {pnum}")
```

Comment (LaPetiteSouris) [reply python_liftbridge/python_liftbridge.py:145]: Agree. This confusion is a bug in the original code itself.

I suggest:

1. In `Stream`, add a new param in the constructor, called `subscribe_to_partition` with default value is 0.

2. Default value for `partitions` in `Stream` should be 1 , indicating that at least 1 partition is created.

3.  In `_subscribe_request`, use `subscribe_to_partition`, in `_create_stream_request`, use `partitions`.

4. Update README

Better explicit than implicit. What do you think ?

Comment (vkudyushev) [reply python_liftbridge/python_liftbridge.py:145]: seems like it's better. check new commit

**Pull request merged**